packages/react-cli/src/commands/generate.ts

```diff
@@ -16,17 +16,15 @@ export class GenerateCommand extends Command {
     '$ react generate route route-name',
     '$ react generate route path/to/route-name',
 
-    '$ react generate component component-name --route=dashboard/posts'
+    '$ react generate component component-name --route=dashboard/posts',
```

Comment:
what about moving this up above the route messages with component since it's a component command.

Reply:
that makes sense! will do